.github/workflows/update-cpp-sdk-on-release.yml

```diff
@@ -35,6 +35,25 @@ jobs:
           private_key: ${{ secrets.CPP_WORKFLOW_TRIGGER_APP_PRIVATE_KEY }}
           repository: firebase/firebase-cpp-sdk
 
+      - name: Ensure Cocoapods repo has been updated
+        run: |
+          # If the new version is simply vX.Y.Z, wait up to an hour for its podspec to be present.
+          if [[ "$GITHUB_REF" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            podspec_url="https://github.com/CocoaPods/Specs/blob/master/Specs/0/3/5/Firebase/${GITHUB_REF:1}/Firebase.podspec.json"
```

Comment:
FYI it may still take additional time to get from the GitHub repo to the CDN

Reply:
How long?

Reply:
(Should I have this check the repo downloaded via pod repo update?)

Reply:
Oh, actually, all of the podspecs we use to build C++ have `source 'https://github.com/CocoaPods/Specs.git'` in them, does that mean they will bypass the CDN?

Reply:
Yep, they'll be slower, but bypass the CDN in that case. So this flow could be fine